config/deploy.rb

```diff
@@ -51,4 +51,4 @@
 
 after "deploy:update", "deploy:cleanup"
 after "deploy:symlink", "deploy:link_folders"
-after "deploy:artisan_migrate", "deploy:artisan_cache_clear"
+after "deploy:link_folders", "deploy:artisan_migrate"
```

Comment:
Yeah, it’d be nice to have a “clean” starter Wercker & Cap config files. We still have some strange public folder linking step that I can't seem to remove that tries to do something to a `public/images` and `public/stylesheets` folder on every Northstar deploy. :grimacing: